NEXTCLOUD.cmake

```diff
@@ -69,6 +69,16 @@ if(WIN32)
     # MSI Upgrade Code (without brackets)
     set( WIN_MSI_UPGRADE_CODE                "FD2FCCA9-BB8F-4485-8F70-A0621B84A7F4" )
 
+    # CfAPI Shell Extensions
+    set( CFAPI_SHELL_EXTENSIONS_LIB_NAME CfApiShellExtensions )
```

Comment:
@mgallien What do you mean to `overwrite`? The variable is here because we need to know the proper DLL name when registering it in runtime. Otherwise, if we just use the hardcoded name everywhere, should you change its name at least in one place, and, you will have either compilation is broken (best case scenario), or, the shell extension not working and Explorer freezing for like a minute, if the DLL name registered is incorrect and the DLL does not exist.